lib/Minion/Backend/Pg.pm

```diff
@@ -267,21 +267,48 @@ sub _try {
   my ($self, $id, $options) = @_;
 
   return $self->pg->db->query(
-    q{UPDATE minion_jobs SET started = NOW(), state = 'active', worker = ?
-      WHERE id = (
-        SELECT id FROM minion_jobs AS j
-        WHERE delayed <= NOW() AND id = COALESCE(?, id) AND (parents = '{}' OR NOT EXISTS (
-          SELECT 1 FROM minion_jobs WHERE id = ANY (j.parents) AND (
-            state = 'active' OR (state = 'failed' AND NOT j.lax)
-            OR (state = 'inactive' AND (expires IS NULL OR expires > NOW())))
-        )) AND priority >= COALESCE(?, priority) AND queue = ANY (?) AND state = 'inactive' AND task = ANY (?)
-          AND (EXPIRES IS NULL OR expires > NOW())
-        ORDER BY priority DESC, id
-        LIMIT 1
-        FOR UPDATE SKIP LOCKED
-      )
-      RETURNING id, args, retries, task}, $id, $options->{id}, $options->{min_priority},
-    $options->{queues} || ['default'], [keys %{$self->minion->tasks}]
+      q{
+        -- Set lock timeout to prevent long waits (50ms)
```

Comment:
The commend doesn't really add any information the code doesn't already contain.